src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -503,11 +514,11 @@ public override void VisitMethod(IMethodSymbol symbol)
 
                             if (sourceUserDefinedOperatorSymbolBase is SourceUserDefinedConversionSymbol)
                             {
-                                addUserDefinedConversionName(symbol, operatorName);
+                                addUserDefinedConversionName(symbol, tryGetUserDefinedConversionTokenKind(operatorName), operatorName);
```

Comment:
There is an implicit dependency here that `tryGetUserDefinedConversionTokenKind` does not return `SyntaxKind.None`. Why is it safe to assume that on this code path but other code paths explicitly handle its return?

Reply:
> Why is it safe to assume that on this code path but other code paths explicitly handle its return?

This code path handles C# symbols (see ```is SourceUserDefinedOperatorSymbolBase``` check above, for those the helper is expected to always succeed.